pennylane/templates/subroutines/qsvt.py

```diff
@@ -0,0 +1,279 @@
+
+
+    And when the number of projector-controlled phase shifts is odd (:math:`d` is even):
+
+    .. math::
+
+        U_{QSVT} = \left[\prod^{d/2}_{k=1}\Pi_{\phi_{2k-1}}U^\dagger\Pi_{\phi_{2k}}U\right]
+        \Pi_{\phi_{d+1}}.
+
+    This circuit applies a polynomial transformation (:math:`Poly^{SV}`) to the singular values of
+    the block encoded matrix:
+
+    .. math::
+
+        \begin{align}
+             U_{QSVT}(A, \vec{\phi}) &=
+             \begin{bmatrix}
+                Poly^{SV}(A) & \cdot \\
+                \cdot & \cdot
+            \end{bmatrix}.
+        \end{align}
+
+    .. seealso::
+
+        :func:`~.qsvt` and `A Grand Unification of Quantum Algorithms <https://arxiv.org/pdf/2105.02859.pdf>`_
+
+    Args:
+        UA (Operator): the block encoding circuit, specified as an :class:`~.Operator`,
+            like :func:`~.BlockEncode`
+        projectors (Sequence[Operator]): a list of projector-controlled phase
+            shifts that implement the desired polynomial
+        wires (Iterable): the wires the template acts on
+
+    Raises:
+        ValueError: if the input block encoding is not an operator
+
+    **Example**
+
+    >>> dev = qml.device("default.qubit", wires=2)
+    >>> A = np.array([[0.1]])
+    >>> block_encode = qml.BlockEncode(A, wires=[0, 1])
+    >>> angles = [qml.PCPhase(i + 0.1, dim=1, wires=[0, 1]) for i in range(3)]
+    >>> @qml.qnode(dev)
+    >>> def example_circuit():
+    ...    qml.QSVT(block_encode, angles, wires=[0, 1])
+    ...    return qml.expval(qml.PauliZ(wires=0))
+    >>> print(qml.draw(example_circuit)())
+    0: ─╭QSVT─┤  <Z>
+    1: ─╰QSVT─┤
+    >>> print(qml.draw(example_circuit,expansion_strategy='device')())
+    0: ─╭∏_ϕ(0.10)─╭BlockEncode(M0)─╭∏_ϕ(1.10)─╭BlockEncode(M0)†─╭∏_ϕ(2.10)─┤  <Z>
+    1: ─╰∏_ϕ(0.10)─╰BlockEncode(M0)─╰∏_ϕ(1.10)─╰BlockEncode(M0)†─╰∏_ϕ(2.10)─┤
+    """
+
+    num_wires = AnyWires
+    """int: Number of wires that the operator acts on."""
+
+    grad_method = None
+    """Gradient computation method."""
+
+    def __init__(self, UA, projectors, wires, do_queue=True, id=None):
+        if not isinstance(UA, qml.operation.Operator):
+            raise ValueError("Input block encoding must be an Operator")
+
+        self._hyperparameters = {
+            "UA": UA,
+            "projectors": projectors,
+        }
+
+        super().__init__(wires=wires, do_queue=do_queue, id=id)
+
+    @staticmethod
+    def compute_decomposition(
+        UA, projectors, **hyperparameters
+    ):  # pylint: disable=arguments-differ
+        r"""Representation of the operator as a product of other operators.
+
+        The :class:`~.QSVT` is decomposed into alternating block encoding
+        and projector-controlled phase shift operators. This is defined by the following
+        equations, where :math:`U` is the block encoding operation and both :math:`\Pi_\phi` and
+        :math:`\tilde{\Pi}_\phi` are projector-controlled phase shifts with angle :math:`\phi`.
```

Comment:
I don't see `:math:\tilde{\Pi}_\phi` mentioned elsewhere.

Reply:
Added the tilde where it was missing in the equations below and the other places where this equation appears.